frontend/services/entities/mdm.ts

```diff
@@ -46,6 +46,10 @@ export interface IUploadProfileApiParams {
   labels?: string[];
 }
 
+export const isDDMProfile = (name: string) => {
+  return name.includes(".json");
```

Comment:
This is the right test to apply during the upload modal flow. But for the profiles list and other places, I think the better approach would be to check the `profile_uuid` to see if it starts with "d". I don't think the `name` field that is stored in the DB and returned by the API will include `.json`. 

@roperzh does that line up with your understanding?

Reply:
ah damn, sorry @ghernandez345 I told you that you could use the extension, but I think Sarah is right.

Reply:
updated